query_V1

```diff
@@ -11,43 +26,154 @@
 		<link rel="stylesheet" href="css/style_query.css">
 		
 		<style>
-			.row:nth-child(even) {background-color: white;}
+			
+			@import url('https://fonts.googleapis.com/css2?family=Noto+Sans+JP:wght@700;900&display=swap');
+			
+			html {
+				overflow-x: hidden;
+				scroll-behavior: smooth;
+			}
+			
+			.bottom-to-top:hover img{transform: scale(1.1, 1.1); transition: 0.6s;}
+			.bottom-to-top, img {transition: 0.6s;}
+			
+			.grid-container {
+				display: grid;
+				grid-template-columns: 20vw 80vw;
+				grid-template-rows: 10vw 8vw 245vw 10vw;
+			}
+			
```

Comment:
This is some styling code that is used to style table that contains the query in my page.